code/Hieu_L/python/classes_lab.py

```diff
@@ -0,0 +1,54 @@
+import random
+class bank_account:
+    def __init__(self, name, balance, account_number):
+        self.name = name
+        self.balance = balance
+        self.account_number = random.randint(0,9)
+    def __repr__(self):
+        return print(f"account number: {self.account_number} \n name: {self.name} \n account number: {self.account_number}")
+
+    def deposit(self):
+        amount = input('enter deposit amount: ')
+        self.balance += amount
+        return self.balance
+    
+    def withdrawal(self):
+        amount = input('enter withdrawal amount: ')
```

Comment:
Let's not ask for the input here. Instead , the deposit function should have two arguments:

def deposit(self, amount):
        self.balance += amount
        return self.balance